regulations/static/regulations/css/scss/partials/_drawer.scss

```diff
@@ -64,6 +64,7 @@ Drawer default state
   background-color: $white;
   border-right: 2px solid $panel_border_color;
   border-bottom: 1px solid $panel_border_color;
+  z-index: 100;
```

Comment:
Is 100 _**enough**_?

Reply:
![](https://www.myinstants.com/media/instants_images/9000Techno.gif)